core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -286,6 +286,33 @@ private[spark] class TaskSchedulerImpl(
     }
   }
 
+  override def completeTasks(
+    partitionId: Int, stageId: Int, taskInfo: TaskInfo, killTasks: Boolean): Unit = {
```

Comment:
nit: double indent the params (4 space), each on their own line)

Reply:
Done, not reqd anymore